src/parse-stream.js

```diff
@@ -567,6 +567,51 @@ export default class ParseStream extends Stream {
         this.trigger('data', event);
         return;
       }
+      match = (/^#EXT-X-DATERANGE:(.*)$/).exec(newLine);
```

Comment:
My understanding from reading the spec is that there could be multiple `DATERANGE` tags. Will this code support multiple tags?

Reply:
Just pushed a change to support multiple tags!